Makefile

```diff
@@ -72,7 +72,7 @@ verify: ## Run verifications. Example: make verify
 .PHONY: verify
 
 build: ## Build the executable. Example: make build
-	go build -a -mod=vendor -race -ldflags $(LDFLAGS) -o _output/csi-driver-shared-resource ./cmd
```

Comment:
could you add a `build-race` make target that still let's one do the race tests when they are on platforms that support it @adambkaplan

Reply:
I just sent a [small PR to tackle `-race` flag](https://github.com/adambkaplan/openshift-csi-driver-shared-resource/pull/1) on different platform, so we can keep a single `build` target on the `Makefile`. Please consider.

Reply:
oooh even better :-) ...... thanks @otaviof 

and got to admit, I employed such platform checks quite a bit in my now distant past, programming in C and C++, and totally forgot about them; in my defense, @otaviof is much younger than me and undoubtedly has a better memory :-)

Reply:
Thanks, Gabe! Granted some "creative stealing" to find a good example out there. :-)

Reply:
done!